cmd/format/format.go

```diff
@@ -197,8 +197,8 @@ func Run(v *viper.Viper, statz *stats.Stats, cmd *cobra.Command, paths []string)
 		return fmt.Errorf("failed to close walker: %w", err)
 	}
 
-	// print stats to stdout, unless we are processing from stdin and therefore outputting the results to stdout
-	if !cfg.Stdin {
+	// print stats to stdout unless we are processing from stdin or verbosity level is lower than info (1)
+	if !cfg.Stdin && cfg.Verbose > 0 {
 		statz.Print()
```

Comment:
It might make sense to send the stats to stderr since it's a form of reporting. Stdout is typically reserved for data that can be further processed. And treefmt is already using stdout to print formatted output on `--stdin` here. Sending the statz to stderr would allow to drop the `!cfg.Stdin` check as well.